public/index.html

```diff
@@ -24,6 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <script src="https://kit.fontawesome.com/b4b59ecaeb.js"></script>
```

Comment:
I was wondering if this imports only one, delete icon?

Reply:
It's a new way to use font awesome [font awesome kits](https://blog.fontawesome.com/introducing-font-awesome-kits)
Not sure if this only import the delete icon, but I have checked the network tab and there're some requests for CSS files!!
Anyway they say this way is faster so we're just going to trust them

Reply:
Yep, I've never used font awesome with kit. 
ah, this link is helpful. It says, `Automatic Subsetting! Speed up your web sites by loading only the icons your site is using.` cool cool! 👌🏻

Reply:
Oh I haven't notice this line 
Nice!